### DISCUSSION TITLE
how to know which version type  bump it is?

### DISCUSSION THREAD

**Discussion by RazaGR** [GitTools/GitVersion#3809] [Q&A] [answered]
right now I am using steps.gitversion.outputs.semVer by comparing theprevious version to know if its a major, minor or patch update. is there any better way to know what kind of version bump is this?

**Comment by asbjornu** [comment, marked as answer]
None that I can think of, but it's a good idea for a feature request. GitVersion knows internally which version it has bumped, so it shouldn't be too much work to expose it in an output variable.

**Reply by RazaGR** [reply]
Thanks, posted feature request https://github.com/GitTools/GitVersion/issues/3812
